runtime/autoload/health/provider.vim

```diff
@@ -502,7 +502,14 @@ function! s:check_node() abort
           \ ['Install Node.js and verify that `node` and `npm` commands work.'])
     return
   endif
-  call health#report_info('Node: '. s:system('node -v'))
+  let node_v = get(split(s:system('node -v'), "\n"), 0, '')
+  call health#report_info('Node: '. node_v)
+  if !s:shell_error && s:version_cmp(node_v[1:], '6.0.0') < 0
```

Comment:
What if there is a s:shell_error?

Reply:
I assume that `node -v` failed (how?) so I can't rely on its output or its behaviour when running `neovim-node-host`. 
I'd rather (add a warning and) return early here but the existing code continues on.

`provider#node#can_inspect` handles this case by returning early.